packages/zilliqa-js-crypto/src/index.ts

```diff
@@ -16,30 +16,7 @@
 //  along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 import * as schnorr from './schnorr';
-
-// This is a workaround. We need to improve it.
-const Signature = require('elliptic/lib/elliptic/ec/signature');
-// Q. Why do we use require() here?
-// A. At the moment, Signature() in 'elliptic' is can only be imported
-// from 'elliptic/lib/elliptic/ec/signature'. If we use ES6 import syntax,
-// TS will try to generate d.ts from 'elliptic/lib/elliptic/ec/signature'
-// causing an error with 'Could not find a declaration file for module ...' message.
-// Therefore, we use require() here.
-//
-// Q. Why don't we use `ec.Signature` type for 'Signature()'?
-// A. `Signature()` is a function while @types/elliptic defines it as following
-//
-// class Signature {
-//   r: BN;
-//   s: BN;
-//   recoveryParam: number | null;
-//   constructor(options: SignatureInput, enc?: string);
-//   toDER(enc?: string | null): any; // ?
-// }
-//
-// With the above `ec.Signature` type, `new Signature()` will throw an error
-// since it doesn't have constructor().
-// Therefore, we keep the type of Signature() as any.
+import Signature from 'elliptic/lib/elliptic/ec/signature.js'
```

Comment:
I fixed this by adding the declarations inside `typings/signature.d.ts` so the error is not thrown anymore. The pull request includes that.

Reply:
Yes but the `typings/signature.d.ts` is not in the build output.

Reply:
Ah ok, can it be added to the build output? I'm thinking the best way to work this out is to write our own custom declarations and use it as it would solve the above error.

Reply:
You can read this
https://stackoverflow.com/questions/56018167/typescript-does-not-copy-d-ts-files-to-build

Reply:
Alright so then there are two options i presume? Either declare it as a typescript file or add a build step that copies the declarations. Thank you for your patience 😅

Reply:
`declare it as a typescript file or add a build step` -> `declare it as a typescript file and add a build step`
That is one option, not two.